corehq/apps/hqadmin/management/utils.py

```diff
@@ -54,18 +62,46 @@ def _get_pr_info(pr_number):
 
     line_changes = additions + deletions
 
+    labels = _get_pr_labels(pr_number)
+
     return {
         'number': json_response['number'],
         'title': json_response['title'],
+        'body': json_response['body'],
         'opened_by': json_response['user']['login'],
         'url': json_response['html_url'],
         'merge_base': json_response['base']['label'],
         'merge_head': json_response['head']['label'],
         'additions': additions,
         'deletions': deletions,
         'line_changes': line_changes,
+        'labels': labels,
     }
 
+
+def _get_pr_labels(pr_number):
+    url = 'https://api.github.com/repos/dimagi/commcare-hq/issues/{}'.format(pr_number)
+    json_response = requests.get(url).json()
+    if not json_response.get('number'):
+        # We've probably exceed our rate limit for unauthenticated requests
+        return []
+    assert pr_number == json_response['number'], (pr_number, json_response['number'])
+
+    return [
+        {'name': label['name'], 'color': label['color']}
+        for label in json_response['labels']
+    ]
+
+
+def _get_prs_by_label(pr_infos):
+    prs_by_label = {}
+    for label in LABELS_TO_EXPAND:
```

Comment:
nit: this is fine, but you run through each PR multiple times here.
another way of doing this could be:
```py
from collections import defaultdict
prs_by_label = defaultdict(list)
for pr in pr_infos:
    for label in pr['labels']:
        if label['name'] in EXPAND_LABELS:
            prs_by_label[label['name']].append(pr)
```